terraform/variables.tf

```diff
@@ -10,6 +10,34 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+variable "production_bucket_name" {
+  type        = string
+  description = "The name of the S3 bucket where the production Assessor Portal tarball lives."
+  default     = "cisa-cool-third-party-production"
+}
+
+variable "production_objects" {
+  type        = list(string)
+  description = "The Assessor Portal tarball object inside the production bucket."
+  default = [
+    "assessor-portal-python-v1.1.1_test.tgz",
+  ]
+}
```

Comment:
You should either change the variable name and type to make it singular or modify the description to indicate plurality.

Reply:
I was being lazy!  Resolved in https://github.com/cisagov/ansible-role-assessor-portal/pull/3/commits/d86a98446ebf9b90b6a314befeb6384a7d46a399.